### ISSUE TITLE
Black indent lines

### ISSUE THREAD

**Issue by ebramirez** [jaredgorski/Styrokai#3]
The indent lines are not visible because they are the same color as the background.

Here is an example:
![Screenshot from 2019-07-24 22-36-01](https://user-images.githubusercontent.com/42619854/61826806-79cd3e00-ae63-11e9-8746-395f34925b5d.png)

Changing it to gray would be great.

**Comment by ebramirez** [comment]
In the meantime, a temporary solution is to add this to the `settings.json`:

```javascript
"workbench.colorCustomizations": {
        "[Styrokai]": {
            "editorIndentGuide.background": "#777777",
            "editorIndentGuide.activeBackground": "#ffffff"
        }
    }
```

**Comment by jaredgorski** [comment]
Technically closed via #4. Note that this will not automatically affect the change on Visual Studio Marketplace; I will need to update it there. Since I am on my honeymoon, this may be delayed. Due to this, leaving open till the update is pushed to VSM.

**Comment by jaredgorski** [comment]
Closed by update to version 0.2.0
